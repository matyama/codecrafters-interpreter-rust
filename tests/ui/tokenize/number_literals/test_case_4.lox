(5+3) > 7 ; "Success" != "Failure" & 10 >= 5
