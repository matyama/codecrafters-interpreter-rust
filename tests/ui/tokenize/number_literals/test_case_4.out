LEFT_PAREN ( null
NUMBER 5 5.0
PLUS + null
NUMBER 3 3.0
RIGHT_PAREN ) null
GREATER > null
NUMBER 7 7.0
SEMICOLON ; null
STRING "Success" Success
BANG_EQUAL != null
STRING "Failure" Failure
[line 1] Error: Unexpected character: &
NUMBER 10 10.0
GREATER_EQUAL >= null
NUMBER 5 5.0
EOF  null
